pkg/netceptor/netceptor.go

```diff
@@ -199,11 +200,12 @@ const (
 
 // ServiceAdvertisement is the data associated with a service advertisement
 type ServiceAdvertisement struct {
-	NodeID   string
-	Service  string
-	Time     time.Time
-	ConnType byte
-	Tags     map[string]string
+	NodeID       string
+	Service      string
+	Time         time.Time
+	ConnType     byte
+	Tags         map[string]string
+	WorkCommands []string
```

Comment:
I may expand this PR further to include some capacity data in this struct as part of the advertisements.  We already track this data per receptor and make it available via the status command, but I suppose we could just as easily attach it here.

Reply:
After chatting w/ @matburt, I think capacity and versioning don't belong here, because they're not really concepts below the mesh/at the netceptor level.  We'll revisit how we want to deal with this as we continue to experiment with integration w/ AWX in the coming weeks.